utils/constants.py

```diff
@@ -17,12 +17,29 @@
     "data.end_local_dt",
     "data.end_fmt_time",
     "data.duration",
+    "data.duration.raw",
     "data.distance",
+    "data.distance.raw",
     "data.start_loc.coordinates",
     "data.end_loc.coordinates",
     "user_id"
 ]
 
+INITIAL_TRIP_COLS =[
```

Comment:
why are we essentially duplicating `VALID_TRIP_COLS` here? I don't love the listing out of valid cols (so many hardcoded lists), but there is really no reason to have two almost identical lists.

Also, for raw, we should just provide them with everything that we have in the table. No filtering.